src/subscript/field_statistics/field_statistics.py

```diff
@@ -1291,17 +1409,19 @@ def calc_stats(
                             sum_total_active = np.ma.sum(sum_active) / nreal
                             sum_total_code = np.ma.sum(number_of_cells) / nreal
                             fraction = sum_total_code / sum_total_active
-                            txt1 = f"Average number of active cells: {sum_total_active}"
+                            txt1 = (
+                                f"  Average number of active cells: {sum_total_active}"
+                            )
```

Comment:
I don't see txt1, txt2, txt3 and txt4 being used anywhere else. Would it be sufficient that you put it's content directly into logger.info?

Reply:
I can do that.